exercises/random/challenge-056.py

```diff
@@ -0,0 +1,34 @@
+### RANDOM CHALLENGE ###
+
+"""
+056
+Randomly pick a whole number between 1 and 10. 
+Ask the user to enter a number and keep entering numbers 
+until they enter the number that was randomly picked.
+"""
+
+# I have added a function that is reused across exercises
+import common_functions
+import random
+
+computer_choice = random.randint(1, 10)
+
+# to avoid the break statement, I have encapsulated the logic into a function
```

Comment:
```suggestion
# to avoid the break statement, I have encapsulated the logic into a function
```
When someone reads this code they will be unaware of any break statement so this comment is only useful to you